fe/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Flex,
+  Box,
+  Input,
+  Text,
+  FormLabel,
+  Textarea,
+  Button,
+} from '@chakra-ui/react';
+import {
+  Formik,
+  Form,
+} from 'formik';
+
+interface MyFormValues {
+  name: string;
+  description: string;
+}
+
+function ProductPage(): JSX.Element {
+  const initialValues: MyFormValues = { name: '', description: '' };
+  return (
+    <Flex direction="column" align="center">
+      <Text fontSize="30px" mb="30px">Product</Text>
+      <Box align="center">
+        <Formik
+          initialValues={initialValues}
```

Comment:
There should be form `validation` here as well, so the user could not submit an empty Product.
At least the `Name` input should be `required` in order to create a new product.